lib/src/radar_chart/radar_chart.dart

```diff
@@ -85,7 +87,9 @@ class RadarChart extends StatefulWidget {
       this.animate = true,
       this.animationDuration = const Duration(milliseconds: 1500),
       this.curve = Curves.easeIn,
-      this.chartRadiusFactor = 0.8});
+      this.chartRadiusFactor = 0.8,
+      this.showPoint = false,
+      this.pointRadius = 0.0});
```

Comment:
default value of pointRadius should not be 0.0, keep something smaller value

Reply:
Do you mean minus(-) value? @intkhabahmed

Reply:
In 'radar_chart_painter.dart'
0.0 value is depend on screen size

Reply:
Should be greater than 0 by default